client/src/bastionlab/polars/client.py

```diff
@@ -23,15 +23,13 @@
 
 
 class BastionLabPolars:
-    """
-    Main BastionLabPolars API class.
+    """Main BastionLabPolars API class.
```

Comment:
As the Torch part is now part of BastionLab, this phrase is wrong. I would rather specify that it's the polars related functions, or something like that.

Reply:
I've now updated the description of the BastionLabPolars class according to the integration of torch related functionnalities.

Reply:
Thank you, that's way better. However, the torch client lacks details. Please do the same for the torch client.